src/symbols/symbols_with_dbghelp.cpp

```diff
@@ -449,4 +451,76 @@ namespace dbghelp {
 }
 }
 
+namespace cpptrace {
+namespace experimental {
+/*
+When a module was loaded at runtime with LoadLibrary after SymInitialize was already called,
+it is necessary to manually load the symbols from that module with SymLoadModuleEx.
+
+See "Symbol Handler Initialization" in Microsoft documentation at
+https://learn.microsoft.com/en-us/windows/win32/debug/symbol-handler-initialization
+*/
+void load_symbols_for_file(const std::string& name) {
+    HMODULE module = GetModuleHandleA(name.c_str());
+    if (module == NULL) {
+        std::fprintf(
+            stderr, 
+            "Error: Unable to load symbols for file %s: %s\n", 
+            name.c_str(), 
+            std::system_error(GetLastError(), std::system_category()).what()
+        );
+        return;
+    }
+
+    MODULEINFO moduleInfo;
+    if (0 == GetModuleInformation(
+        GetCurrentProcess(),
+        module,
+        &moduleInfo,
+        sizeof(moduleInfo)
+    )) {
+        std::fprintf(
+            stderr, 
+            "Error: Unable to get module information for %s: %s\n", 
+            name.c_str(), 
+            std::system_error(GetLastError(), std::system_category()).what()
+        );
+        return;
+    }
+
+    auto lock = cpptrace::detail::get_dbghelp_lock();
+    HANDLE syminit_handle = cpptrace::detail::ensure_syminit().get_process_handle();
+    if (0 == SymLoadModuleEx(
+        syminit_handle,
+        NULL,
+        name.c_str(),
+        NULL,
+        (DWORD64)moduleInfo.lpBaseOfDll,
+        moduleInfo.SizeOfImage,
+        NULL,
+        0
+    )) {
+        std::fprintf(
+            stderr, 
+            "Error: Unable to load symbols for file %s: %s\n", 
+            name.c_str(), 
+            std::system_error(GetLastError(), std::system_category()).what()
+        );
+    }
+}
+}
+}
+#else
+
+#include <cpptrace/utils.hpp>
+
+namespace cpptrace {
+namespace experimental {
+
+void load_symbols_for_file(const std::string& name) {
```

Comment:
This whole file is really meant for `CPPTRACE_GET_SYMBOLS_WITH_DBGHELP` stuff, it'd be better to define this for non-windows elsewhere. Alternatively, does this function make sense to have on platforms other than windows?